static/app/views/alerts/rules/issue/ruleNode.tsx

```diff
@@ -423,7 +424,7 @@ function RuleNode({
                 {...deps}
                 link={node.link}
                 ticketType={node.ticketType}
-                instance={data}
+                instance={data as unknown as TicketActionData}
```

Comment:
not sure if there's a preferred way to cast the type...

for reference here is the old type
https://github.com/getsentry/sentry/blob/c1b7e74ac8a1ffd2e9872763c18d4830e0718664/static/app/types/alerts.tsx#L196-L201

vs the new type
https://github.com/getsentry/sentry/blob/9e2a786cbefee284e03602ce0fa017c8e192a48a/static/app/types/alerts.tsx#L210-L214

they are both pretty generic with `[key: string]: any;` 😅